helm-chart/binder-launches/templates/configmap.yaml

```diff
@@ -12,12 +12,8 @@ data:
     {{- if .Values.port }}
     port: {{ .Values.port }},
     {{- end }}
-    {{- if .Values.debug }}
     debug: {{ .Values.debug }},
-    {{- end }}
-    {{- if .Values.db.ssl }}
     dbSSL: {{ .Values.db.ssl }},
```

Comment:
Can we try this?

```suggestion
    {{- if or .Values.db.ssl (eq (.Values.db.ssl | toString) "false") }}
    dbSSL: {{ .Values.db.ssl }},
    {{- end }}
```

Reply:
How about something like?
```
{{ if not (kindIs "invalid" .Values.db.ssl) }} 
```

Reply:
this checks for nil value (and not boolean).

Reply:
Yes, lets try it out :)